helm/charts/pygeoapi/templates/NOTES.txt

```diff
@@ -0,0 +1,22 @@
+1. Get the application URL by running these commands:
```

Comment:
Can we turn this into `README.md` as well?

Reply:
I included some extra helper information - let me know if this is appropriate, otherwise it can be removed if we want to keep it as a strict copy to README.md from NOTES.txt

Reply:
We should remove `NOTES.txt` and put any relevant information in this `README.md`.